ui/apps/everest/src/pages/database-form/database-form-body/steps/first/first-step.tsx

```diff
@@ -278,7 +278,7 @@ export const FirstStep = ({ loadingDefaultsForEdition }: StepProps) => {
           name={DbWizardFormFields.dbVersion}
           label={Messages.labels.dbVersion}
           selectFieldProps={{
-            disabled: mode === 'restoreFromBackup' || mode === 'edit',
+            disabled: mode === 'restoreFromBackup',
```

Comment:
what is this change? Is it indended?

Reply:
Yes, that's intended. By removing `mode === 'edit'`, we allow editing the engine version from the UI